app/src/main/java/amazingme/activities/app/GameSelectionPage.java

```diff
@@ -3,6 +3,7 @@
 import android.view.View;
```

Comment:
all of this is fine for now... be ready to pick it back up (we need to dynamically load the page based on what games are available).